website/content/docs/agent/limits/usage/limit-request-rates-from-ips.mdx

```diff
@@ -8,6 +8,12 @@ description: Learn how to set read and request rate limits on RPC and gRPC traff
 
 This topic describes how to configure RPC and gRPC traffic rate limits for source IP addresses. This enables you to specify a budget for read and write requests to prevent any single source IP from overwhelming the Consul server and negatively affecting the network. For information about setting global traffic rate limits, refer to [Set a global limit on traffic rates](/consul/docs/agent/limits/usage/set-glogal-traffic-rate-limits). For an overview of Consul's server rate limiting capabilities, refer to [Limit traffic rates overview](/consul/docs/agent/limits/overview). 
 
+<EnterpriseAlert>
+
+This feature requires Consul Enterprise. Refer to the [feature compatibility matrix](/consul/docs/v1.16.x/enterprise#consul-enterprise-feature-availability) for additional information.
```

Comment:
since this page only appears in 1.16 (doesn't exist in 1.15) we don't need to explicitly add v1.16.x (if you do it'll break)

```suggestion
This feature requires Consul Enterprise. Refer to the [feature compatibility matrix](/consul/docs/enterprise#consul-enterprise-feature-availability) for additional information.
```